app/models/benchmark_run.rb

```diff
@@ -14,7 +14,7 @@ class BenchmarkRun < ApplicationRecord
   # FIXME: Remove this and order by Commit#created_at
   default_scope { order("#{self.table_name}.created_at DESC")}
 
-  scope :fetch_commit_benchmark_runs, ->(form_result_type, benchmark_result_type, limit) {
+  scope :fetch_commit_benchmark_runs, -> (form_result_type, benchmark_result_type, limit) {
```

Comment:
Really minor thing: if you're just making a one-space change in a given file, maybe best to skip it for now :-)